include/seqan3/alignment/pairwise/policy/all.hpp

```diff
@@ -57,7 +57,7 @@
  * Function name     | Arguments                              | Return value                         |
  * ----------------- | -------------------------------------- | ------------------------------------ |
  * `compute_cell`    | `cell &&`, `cache &`, `score const &`  | `void`                               |
```

Comment:
```suggestion
 * `?compute_cell?`    | `cell &&`, `cache &`, `score const &`  | `void`                               |
```

Reply:
ah yes, this documentation needs to be adapted. But I wanted to do it when I move it to detail since we don't allow subsubdirectories anymore